### DISCUSSION TITLE
phpmailer going to spam

### DISCUSSION THREAD

**Discussion by fullstact69** [PHPMailer/PHPMailer#2241] [General]
I have used phpmailer at my website. My website's hosting is plesk.
When I try to send the mail using phpmailer, I can see the email at spam of gmail, junk of outlook.
I think this is a problem of mail server. What should I do at my hosting? SPF was already added, and DKIM was already checked. Hope your help. Thank you.

**Comment by Synchro** [comment]
There are many reasons this can happen, pretty much all of them discussed in [this wiki page](https://github.com/PHPMailer/PHPMailer/wiki/Improving-delivery-rates,-avoiding-spam-filters).
